packages/ui/lib/toast/transaction-toast.test.tsx

```diff
@@ -67,13 +67,9 @@ describe('TransactionToast', () => {
         'Building Send transaction',
         expect.objectContaining({
           description: (
-            <Progress
-              status='in-progress'
-              background='stone'
-              value={50}
-              size='sm'
-              className='mt-2'
-            />
+            <div className='mt-2'>
```

Comment:
I removed `className` from `<Progress />`'s props, so we need to wrap it in a div with a margin-top.